pkg/apis/rollouts/v1alpha1/analysis_types.go

```diff
@@ -134,6 +134,16 @@ type MeasurementRetention struct {
 	Limit int32 `json:"limit" protobuf:"varint,2,opt,name=limit"`
 }
 
+// TtlStrategy defines the strategy for the time to live depending on if the analysis succeeded or failed
+type TtlStrategy struct {
```

Comment:
NIT: can we adopt golang acronym casing for this type and name this `TTLStrategy` instead?

https://google.github.io/styleguide/go/decisions.html#initialisms

Reply:
Done